pages/_type/_id.vue

```diff
@@ -508,7 +508,10 @@
             <a href="https://atlauncher.com/about" target="_blank"
               >ATLauncher</a
             >
-            and <a href="https://multimc.org/" target="_blank">MultiMC</a>.
+            , <a href="https://multimc.org/" target="_blank">MultiMC</a>
+            and <a href="https://prismlauncher.org" target="_blank"
```

Comment:
Fix Prettier please!

Reply:
I... don't quite get the complains it gives...

Should I change the lines 511 and 512 like this?
```html
, <a href="https://multimc.org/" target="_blank">MultiMC</a> and
<a href="https://prismlauncher.org" target="_blank"
```